### ISSUE TITLE
Aventuria Magic III (German)

### ISSUE THREAD

**Issue by elyukai** [elyukai/optolith-client#92]

**Comment by rfeistenauer** [comment]
Is this rule book still developed? Its one of the most important rule books for every magical character.

**Comment by elyukai** [comment]
> Is this rule book still developed? Its one of the most important rule books for every magical character.

Yes, it is. It requires the most invasive changes to make, though, and, well, I can't develop full-time, that's why it takes that long. But progress is made, although not really visible in the issues, I'm afraid.

**Comment by arnohakk** [comment]
Hey, any news on this? I would be willing to help implementing :)

**Comment by arnohakk** [comment]
Hey, any news on this? I would be willing to help implementing.

**Comment by elyukai** [comment]
> Hey, any news on this? I would be willing to help implementing :)

I’m still working on a database rework. A rework was needed due to multiple reasons, but it takes way longer than expected. You can view the current state under https://github.com/elyukai/optolith-database-schema. There are still some missing parts for the schemas (I’m considering *all* books since AMA3 for the rework) and I didn’t track Royals Erben yet, which also causes some database schema changes.

Given the missing schemas, a huge part of the new database contents is already inserted, which is especially obvious in https://github.com/elyukai/optolith-client/projects/7.

Currently needed work goes down to:

- Finishing database schema
- Register the last few books as issues
- Insert missing content as stated by issues

I will rework the client code once the database is ready, since there may still be changes and I cannot work on both at the same time anyway.

Concerning collaboration areas, I think it would still be best to help with database contents. The database schema would require a lot of conversation, I assume, but I don’t know when I will have time to have deep conversations about this, which would be great for collaboration on that. Registering new book issues is something I kind of already do when I read preview PDFs, I just need to convert my notes into issues. And since new database *content* is something, that is not too difficult, but takes a lot of time, helping there is the most time-efficient and -effective way of helping right now, I think.

I hope this helps! And if you want to help with the database, please join the [Optolith Discord server](https://discord.gg/wfdgB9g) (for real-time communication) and tell me! Then I’ll add you to the data repository.

**Comment by arnohakk** [comment]
Sounds great! As soon as I return from my vacation from the Northern Lands I will contact you via discord :)
